### PR TITLE
Make tables sortable

### PR DESCRIPTION
**Pull request by jamesmcclain**
The tables on the "analyze" page and the "water quality" tab of the TR-55 page have been made sortable.

Connects #729

Thanks to @kdeloach for help understanding/debugging the behavior of the `bootstrap-table` plugin.

**To Test**
   * Go to the "analyze" page or the "water quality" section of the TR-55 page.
   * Using the new arrows in the head-row of the tables to (re)sort the tables.

screenshot from 2015-09-01 12 41 42

screenshot from 2015-09-01 12 41 47

screenshot from 2015-09-01 12 41 51

screenshot from 2015-09-01 12 41 55

screenshot from 2015-09-01 12 42 08

screenshot from 2015-09-01 12 42 25

screenshot from 2015-09-01 12 42 29

### CODE REVIEW

File: src/mmw/js/src/core/setup.js [hunk 1]
```diff
@@ -22,6 +22,7 @@ require('../analyze/filters');
 
 require('bootstrap');
 require('bootstrap-select');
+require('bootstrap-table/dist/bootstrap-table.js');
```
Comment (kdeloach) [inline src/mmw/js/src/core/setup.js]: Were you able to verify if `bootstrap-table/dist/bootstrap-table.js` can be added to the vendor bundle?

Comment (jamesmcclain) [inline src/mmw/js/src/core/setup.js]: You mean in the bundle script?  Let me check.

**Comment by rajadain** [comment]
It seems like `bootstrap-table` was added to `npm-shrinkwrap.json` but not `package.json`

**Comment by jamesmcclain** [comment]
Nice catch, thank you.

**Comment by rajadain** [comment]
Could we reduce the padding (green) in the table headers?

**Comment by rajadain** [comment]
Here's what it looks like with 0 vertical padding (horizontal padding is same as before: `0.4rem`):

image

It would also be really cool if sorting the table also sorted the graph, but that might be outside the scope of this card.

**Comment by jamesmcclain** [comment]
> Here's what it looks like with 0 vertical padding (horizontal padding is same as before: 0.4rem):

If you can make a pull request against this branch and/or send me a patch, I will include it.

File: src/mmw/package.json [hunk 2]
```diff
@@ -23,6 +23,7 @@
     "blueimp-md5": "1.1.0",
     "bootstrap": "3.3.4",
     "bootstrap-select": "git://github.com/azavea/bootstrap-select",
+    "bootstrap-table": "^1.8.1",
```
Comment (lliss) [inline src/mmw/package.json]: We've been pegging versions by removing the ^

File: src/mmw/npm-shrinkwrap.json [hunk 3]
```diff
@@ -44,6 +44,844 @@
       "from": "bootstrap-select@git://github.com/azavea/bootstrap-select",
       "resolved": "git://github.com/azavea/bootstrap-select#f1755efa102a41e43fc367ba36ce905b8f2b90e1"
     },
+    "bootstrap-table": {
```
Comment (lliss) [inline src/mmw/npm-shrinkwrap.json]: I can't believe it has this many dependencies.

Comment (jamesmcclain) [inline src/mmw/npm-shrinkwrap.json]: Do you mean that literally or figuratively?

Comment (caseypt) [inline src/mmw/npm-shrinkwrap.json]: It really has none, but they didn't use `devDependencies` in the package.json: https://github.com/wenzhixin/bootstrap-table/blob/master/package.json#L9-L16

**Comment by lliss** [comment]
Tested. All is working as expected. Code looks good. +1

File: src/mmw/npm-shrinkwrap.json (hunk 3 above)
Comment (caseypt) [inline src/mmw/npm-shrinkwrap.json]: Would be a good, quick PR.

**Pull request merged**